### DISCUSSION TITLE
Can you run Lavalink on Pterodactyl

### DISCUSSION THREAD

**Discussion by crmacca** [lavalink-devs/Lavalink#696] [Q&A] [answered]
Hey I was wondering if you can host lavalink via the pterodactyl panel, it would be very useful for people wishing to host bots on my server to be able to host their lavalink nodes in a place they can manage it.

Thanks :)

**Comment by topi314** [comment, marked as answer]
sure you can
I have no experience with pterodactyl tho
there is a prebuilt docker image

**Reply by crmacca** [reply]
Alrighty, I'll attempt to give it a shot!
